modules/kms_multi_region_key/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "kms_key_arn" {
+  value = aws_kms_key.primary.arn
+}
+
+output "kms_key_replica_arn" {
```

Comment:
why is this needed?

Reply:
You can have a replica key as a backup when configuring sops:
```
---
creation_rules:
- path_regex: \.test\.yaml$
  kms: arn:aws:kms:us-east-1:021589213323:key/mrk-fa874364e1964213a6827776fcdb472f
  # kms: arn:aws:kms:us-west-2:021589213323:key/mrk-fa874364e1964213a6827776fcdb472f
  aws_profile: nulogy-go-test
```